muswarmadmin/pipelines.py

```diff
@@ -139,8 +139,6 @@
                                  cwd="/data/%s" % project_id)
     if proc.returncode is not 0:
         await app.update_state(project_id, SwarmUI.Error)
-    else:
-        await app.update_state(project_id, SwarmUI.Up)
```

Comment:
Maybe add a note that says that the status is expected to be updated by the container events.